lib/src/Base/Stat/HMatrixImplementation.cxx

```diff
@@ -320,19 +348,40 @@ void HMatrixImplementation::factorize(const String& method)
   else if (method != "LU")
     LOGWARN( OSS() << "Unknown factorization method: " << method << ". Valid values are: LU, LDLt, or LLt.");
 
-  try
-  {
-    hmat_factorization_context_t context;
-    hmat_factorization_context_init(&context);
-    context.factorization = fact_method;
-    context.progress = NULL;
-    static_cast<hmat_interface_t*>(hmatInterface_)->factorize_generic(static_cast<hmat_matrix_t*>(hmat_), &context);
-  }
-  catch (std::exception& ex)
-  {
-    // hmat::LapackException is not yet exported
-    throw InternalException(HERE) << ex.what();
-  }
+  // Compute a reasonable regularization factor
+  Scalar lambda = 2.0 * computeApproximateLargestEigenValue() * ResourceMap::GetAsScalar("HMatrix-AssemblyEpsilon");
```

Comment:
Care we should have the **same** value that is contained in https://github.com/openturns/openturns/pull/1295/files#diff-227275030f245b905d3e4ae25d6b9054R815

Reply:
I must have missed something here... I don't see the point!

Reply:
If `HMatrixParameters` contains an `AssemblyEpsilon` different, it might cause troubles.

Reply:
Ok I see your point: I completely overlooked this epsilon_. In fact, it should be removed as the new development is here exactly to provide automatically the best possible value for it, taking into account both the characteristics of the matrix (its largest eigenvalue) and the compression error (ie the level of accuracy one can reasonably expect from the compression)